.gitignore

```diff
@@ -0,0 +1 @@
+/.idea/
```

Comment:
There is no code here, not sure why this is needed. _If_ it is needed, probably using a separate PR would be the right way.

Reply:
Ok, let me open a new PR for that.